### ISSUE TITLE
Errors with safe operator

### ISSUE THREAD

**Issue by tilast** [bbatsov/rubocop#3108]
## Expected behavior
Succesfully passing rubocop

## Actual behavior
Errors in terminal
```
Inspecting 1 file
An error occurred while Lint/DefEndAlignment cop was inspecting app/controllers/concerns/render.rb.
To see the complete backtrace run rubocop -d.
An error occurred while Style/IndentationWidth cop was inspecting app/controllers/concerns/render.rb.
To see the complete backtrace run rubocop -d.
.

1 file inspected, no offenses detected

2 errors occurred:
An error occurred while Lint/DefEndAlignment cop was inspecting app/controllers/concerns/render.rb.
An error occurred while Style/IndentationWidth cop was inspecting app/controllers/concerns/render.rb.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
Mention the following information in the issue report:
0.37.0 (using Parser 2.3.0.7, running on ruby 2.3.0 x86_64-darwin15)
```

## Steps to reproduce the problem
code
```ruby
  params[:fields]
      &.to_unsafe_h
      &.map { |key, value| [key, value.split(',').map(&:strip)] }
      &.to_h
```

## RuboCop version
```
$ rubocop -V
0.37.0 (using Parser 2.3.0.7, running on ruby 2.3.0 x86_64-darwin15)
```

**Comment by segiddins** [comment]
Can you try with the latest rubocop version, 0.39?

**Comment by tilast** [comment]
yeap, I'll try

**Comment by tilast** [comment]
it works with 0.39, thanks :)
